solr/core/src/test/org/apache/solr/response/transform/TestChildDocTransformerHierarchy.java

```diff
@@ -32,7 +32,7 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class TestDeeplyNestedChildDocTransformer extends SolrTestCaseJ4 {
+public class TestChildDocTransformerHierarchy extends SolrTestCaseJ4 {
```

Comment:
These "hierarchy" tests are in a separate class since they require a different schema and configuration to be loaded upon start-up.